event_mail/__manifest__.py

```diff
@@ -1,12 +1,12 @@
-# -*- coding: utf-8 -*-
 # © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     'name': 'Event Mail',
     'summary': 'Mail settings in events',
-    'version': '10.0.1.0.0',
+    'version': '11.0.1.0.0',
     'author': 'Tecnativa, '
+              'Therp BV, '
```

Comment:
I don't think the migration grants you co-authorship. You should be in contributors of course.